app/services/move_events/params_validator.rb

```diff
@@ -14,6 +15,7 @@ class ParamsValidator
 
     def initialize(params)
       @timestamp = params.dig(:attributes, :timestamp)
+      @type = params.dig(:type)
```

Comment:
Looks like Rubocop will soon come looking for these single argument digs, so maybe replace with `params[:type]` ?

https://github.com/rubocop-hq/rubocop/issues/7949

Reply:
👍